deploy/scripts/deployDynatraceOnOpenshift.sh

```diff
@@ -7,15 +7,15 @@ DT_API_TOKEN=$(cat creds_dt.json | jq -r '.dynatraceApiToken')
 DT_PAAS_TOKEN=$(cat creds_dt.json | jq -r '.dynatracePaaSToken')
 
 # Deploy Dynatrace operator
-DT_OPERATOR_LATEST_RELEASE=$(curl -s https://api.github.com/repos/dynatrace/dynatrace-oneagent-operator/releases/latest | grep tag_name | cut -d '"' -f 4)
+DT_OPERATOR_LATEST_RELEASE="0.3.1"
```

Comment:
so when we update in the future we need a newer version of the service. Fine with this approach but we need to make it explicit in the release notes.